stable/ambassador/templates/hpa.yaml

```
@@ -0,0 +1,29 @@
+{{- if not .Values.daemonSet }}
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
```

```suggestion
apiVersion: autoscaling/v2beta2
```
Use the latest HPA version.